### DISCUSSION TITLE
patterns/sharding/

### DISCUSSION THREAD

**Discussion by giscus** [iluwatar/java-design-patterns-vuepress-web#65] [Announcements]
# patterns/sharding/

Intent Sharding pattern means divide the data store into horizontal partitions or shards. Each shard has the same schema, but holds its own distinct subset of the data. A shard is a data store in its own right (it can contain the data for many entities of different types), running on a server acting as a storage node.

https://java-design-patterns.com/patterns/sharding/

**Comment by moutainMan** [comment]
Is your implementation consistent hashing?
